script/enterprise-migrations

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+PATH="~/bin:$PATH"
+eval "$(perl -I ~/perl5/lib/perl5/ '-Mlocal::lib')"
+
+# Depends on PGHOST, PGPASSWORD, PGDATABASE, etc
+createdb
+
+# Use the LOGS_DATABASE_URL we use elsewhere but make it fit the DATABASE_URI format
+# sqitch expects
+sqitch --plan-file db/sqitch-enterprise.plan deploy db:$LOGS_DATABASE_URL
+sqitch --plan-file db/sqitch-enterprise.plan verify db:$LOGS_DATABASE_URL
```

Comment:
Naw, no worries at all! I didn't like this approach at all (ew two files to change everytime?!) but know very little about sqitch so did what I could figure out.